com.github.xournalpp.xournalpp.yaml

```diff
@@ -30,10 +31,10 @@ finish-args:
   - "--filesystem=~/.config/dconf:ro"
   - "--talk-name=ca.desrt.dconf"
   - "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
-  - "--persist=.xournalpp"
+  -  - "--persist=.xournalpp"
```

Comment:
is this how it should be? The hyphen looks suspicious to me.

Reply:
Thanks for the catch. I fixed it.